homeassistant/components/lupusec/config_flow.py

```diff
@@ -0,0 +1,89 @@
+""""Config flow for Lupusec integration."""
+
+import ipaddress
+import logging
+import socket
+
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_HOST, CONF_NAME, CONF_PASSWORD, CONF_USERNAME
+from homeassistant.data_entry_flow import FlowResult
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_HOST): str,
+        vol.Required(CONF_USERNAME): str,
+        vol.Required(CONF_PASSWORD): str,
+        vol.Optional(CONF_NAME): str,
```

Comment:
We only allow names to be set by helpers. We should set the name of the config entry to something the user can recognise if possible, otherwise just a default name

Reply:
Changed to: CONF_FRIENDLY_NAME